src/ServicePulse.Host/app/modules/monitoring/js/endpoint_details.controller.js

```diff
@@ -9,7 +9,9 @@
         serviceControlService,
         monitoringService,
         historyPeriods,
-        formatter) {
+        formatter,
+        $filter) {
```

Comment:
I've never seen `$filter` injected like this, and my google-fu is failing. Do you have a link you used that I could take a look through?

Reply:
Does this apply the `| filter` operation that would normally be applied in HTML binding?

Reply:
Correct, it is to use filter in js code. as then I do:
`$filter('graphduration')(instance.metrics.processingTime)`